app/app.go

```diff
@@ -104,10 +107,18 @@ func (a *App) Start() error {
 	mux := http.NewServeMux()
 	mux.HandleFunc("/api/v1/spans", ungzipWrap(a.handleSpans))
 
+	// Handlers enable CORS preflight check support
+	// If ORIGINS_ALLOWED is unset, Access-Control-Allow-Origin header will never be sent,
+	// triggering a CORS preflight check error in browser that issues the request
+	originsOk := handlers.AllowedOrigins(strings.Split(os.Getenv("ORIGINS_ALLOWED"), ";"))
+	headersOk := handlers.AllowedHeaders([]string{"X-Requested-With", "Content-Type"})
```

Comment:
do you need `Content-Type` here? it's one of the headers that's supposed to be allowed without a preflight check - will it not be included in the headers passed to the handler without it?